src/components/OrgView/GlobalSettingsButton.js

```diff
@@ -0,0 +1,185 @@
+import React, { useRef, useState, useCallback } from 'react'
+import PropTypes from 'prop-types'
+import {
+  ButtonBase,
+  ButtonIcon,
+  GU,
+  IconSettings,
+  Popover,
+  useTheme,
+} from '@aragon/ui'
+import IconNetwork from './IconNetwork'
+import IconCustomLabels from './IconCustomLabels'
+import IconNotifications from './IconNotifications'
+import IconHelpAndFeedback from './IconHelpAndFeedback'
+
+function GlobalSettingsButton({ onOpen }) {
+  const theme = useTheme()
+  const [opened, setOpened] = useState(false)
+  const containerRef = useRef()
+
+  const handleToggle = useCallback(() => {
+    setOpened(!opened)
+  }, [setOpened, opened])
+  const handleClose = useCallback(() => {
+    setOpened(false)
+  }, [setOpened])
+  const handleItemClick = useCallback(
+    path => () => {
+      setOpened(false)
+      onOpen(path)
+    },
+    [setOpened, onOpen]
+  )
+
+  return (
+    <React.Fragment>
+      <div ref={containerRef}>
+        <ButtonIcon
+          onClick={handleToggle}
+          css={`
+            height: 100%;
+            width: 40px;
+          `}
+          label="Global preferences"
```

Comment:
Shouldn’t this button (and the corresponding title) be called “Preferences”? Or do we want to indicate to the user that they are applying at a global level? /cc @dizzypaty